projects/microphysics/train/gscond-only-tscale.yaml

```diff
@@ -0,0 +1,100 @@
+#https://wandb.ai/ai2cm/microphysics-emulation/groups/dqc-precpd-limiter-rnn-limited-qc-87e803-30d-v1-online/workspace?workspace=user-aperkins
```

Comment:
note to self: Do we need gscond-only and gscond-tscale...would be nice to choose the better config if possible for now.